### PR TITLE
add updated tar links

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

please cp to `pipelines-docs-1.11`, `pipelines-docs-1.12`

Issue:

RHDEVDOCS 5681

Link to docs preview:

https://67665--docspreview.netlify.app/openshift-pipelines/latest/about/op-release-notes#release-notes-1-11-2_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-11.adoc [hunk 1]
```diff
@@ -67,7 +67,7 @@ Before upgrading to the {pipelines-title} Operator 1.11, ensure that you have at
 
 * With this update, you can use the skopeo tool with {tekton-chains} to generate keys, which are used in the `cosign` signing scheme.
 
-* When you upgrade to the {pipelines-title} Operator 1.11, the previous {tekton-chains} configuration will be overwritten and you must set it again in the `TektonConfig` CR.
+* When you upgrade to the {pipelines-title} Operator 1.11, the previous {tekton-chains} configuration will be overwritten and you must set it again in the `TektonConfig` CR.
```
Comment (xenolinux) [inline modules/op-release-notes-1-11.adoc:70]:
```suggestion
* When you upgrade to the {pipelines-title} Operator 1.11, the previous {tekton-chains} configuration is overwritten and you must set it again in the `TektonConfig` CR.
```

Comment (xenolinux) [reply modules/op-release-notes-1-11.adoc:70]: https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc 

> Use active voice and present tense.

**Review by xenolinux — COMMENTED** [review]
I have one suggestion to consider for now. 
I wanted to check the preview. Can you please provide information such as version, preview link, and issue in the PR description box?

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: modules/op-release-notes-1-11.adoc (hunk 1 above)
Comment (joaedwar) [reply modules/op-release-notes-1-11.adoc:70]: Hello @xenolinux, thank you for your review. I'd like to clarify that both sentences mentioned are written in active voice. You might have concerns regarding the use of future tense, as advised against in the IBM style guide, but I think the se of future tense is intentional here. The sentence with "will" suggests that the configuration's overwriting occurs in the future, right after completing the upgrade. In contrast, the sentence with "is" uses present tense, indicating that the overwriting is an immediate result of the upgrade, happening almost at the same time. This is a follow up PR, to a [PR](https://redhat-internal.slack.com/archives/D05UYR89RSQ/p1699543917093769) that was previously reviewed by peer review squad. In that PR stylistic changes were applied.

**Comment by abhatt-rh** [comment]
> I have one suggestion to consider for now. I wanted to check the preview. Can you please provide information such as version, preview link, and issue in the PR description box?
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

LGTM!

**Pull request merged**